gatsby-config.js

```diff
@@ -117,6 +115,23 @@ module.exports = {
           "/test-items/**",
           "/403",
         ],
+        // eslint-disable-next-line consistent-return
+        filterPages: (page, excludedRoute, tools) => {
+          // Return true excludes the path, false keeps it.
+          if (
+            // Exclude non-canonical pages.
+            !page.pageContext.isCanonical ||
+            // Exclude pages marked with "noindex"
+            page.pageContext.noindex ||
+            // Exclude pages that match the "excludes" array. (default condition)
+            tools.minimatch(
+              tools.withoutTrailingSlash(tools.resolvePagePath(page)),
+              tools.withoutTrailingSlash(excludedRoute)
+            )
+          ) {
+            return true;
+          }
+        },
```

Comment:
👍🏼  I assume this is to get around the annoying behavior of graphql erroring on queries when properties don't exist in Storyblok yet